test/pylib/cpp/conftest.py

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
ALL_MODES, DEBUG_MODES, DEFAULT_ARGS - not used in this commit, please move to commit where it is used

contest in lib, ton sure that this si good name. probably need to rename to common_cpp_conftest to something like this

Reply:
Done